source/driver-models/CodalUSB.cpp

```diff
@@ -43,7 +43,7 @@ static uint8_t usb_status = 0;
 // static uint8_t usb_suspended = 0; // copy of UDINT to check SUSPI and WAKEUPI bits
 static uint8_t usb_configured = 0;
 
-static const ConfigDescriptor static_config = {9, 2, 0, 0, 1, 0, USB_CONFIG_BUS_POWERED, 250};
+static const ConfigDescriptor static_config = {9, 2, 0, 0, 1, 0, USB_CONFIG_BUS_POWERED, 0};
```

Comment:
Probably best to set this to a sensible value (i.e. 50) by default, though I understand it is modified when it is copied into RAM.

Also, why not a #define config? Is that not better for makecode?

Reply:
it sets it to 50 by default, but can by dynamically changed; I imagine it could go into bootloader key in future

Reply:
Okies, just thinking if someone memcpy's that and is unaware of the dynamic nature of the power variable it may cause pain for future us... :)

Reply:
oh, I cleared the value here, so we are not confused as to which is the actual default

Reply:
there are other things there that need to be changed (ie num of interfaces)